scripts/scripts.js

```diff
@@ -201,4 +201,10 @@ const miloLibs = setLibs(LIBS);
     const { default: registerAAUniversity } = await import('./aa-university.js');
     window.addEventListener('mktoSubmit', registerAAUniversity);
   }
+  if (document.querySelector('.faas')) {
+    const link = document.createElement('link');
+    link.setAttribute('rel', 'stylesheet');
+    link.setAttribute('href', '/styles/faas.css');
+    document.head.appendChild(link);
```

Comment:
This is essentially a copy of https://github.com/adobecom/bacom/blob/e0b12f8e125a48e355123d672d421d81c1b58f7d/scripts/scripts.js#L178-L181, could we have a method that is used by both pieces of logic instead?

Reply:
Sounds good, I made a simplified `loadStyle`.